postgres-derive/src/fromsql.rs

```diff
@@ -11,9 +11,36 @@ use crate::overrides::Overrides;
 pub fn expand_derive_fromsql(input: DeriveInput) -> Result<TokenStream, Error> {
     let overrides = Overrides::extract(&input.attrs)?;
 
+    if overrides.name.is_some() && overrides.transparent {
```

Comment:
You should be able to just put this at the bottom of Overrides::extract.

Reply:
I did it this way because I did not know where to take a span / or `dyn ToTokens` to construct a `syn::Error` in `Overrides::extract`. I took inspirations from `serde_derive` which does something similar.

Reply:
Got it, thanks.